docs/querying/sql-functions.md

```diff
@@ -1426,11 +1444,29 @@ If you do not supply an `offset`, returns the value evaluated at the row followi
 
 ## LEAST
 
-`LEAST([expr1, ...])`
+Returns the minimum value from the provided arguments. The expressions must all be convertible to a common data type. 
+
+* **Syntax:** `LEAST([expr1, ...])`
+* **Function type:** Scalar, reduction
+
+<details><summary>Example</summary>
+
+The following example returns the least value between the strings `apple`, `orange` and `pear`, which already share a common datatype.
```

Comment:
"least values" sounds a bit odd. Consider "the minimum value" or " the smallest value".